go/vt/tabletserver/tx_pool.go

```diff
@@ -103,7 +103,7 @@ func (axp *TxPool) TransactionKiller() {
 	defer logError()
 	for _, v := range axp.activePool.GetOutdated(time.Duration(axp.Timeout()), "for rollback") {
 		conn := v.(*TxConnection)
-		log.Warningf("killing transaction: %s", conn.Format(nil))
+		log.Warningf("killing transaction (exceeded timeout: %s): %s", axp.Timeout().String(), conn.Format(nil))
```

Comment:
Yay, nurbileh writing go code :).
Go-ism nit: should be "... timeout: %v)..", axp.Timeout(),...
In go, %v stringifies under the cover.